catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/activity/rtl/RTLOptionMenuIconsTest.kt

```diff
@@ -131,8 +131,8 @@ class RTLOptionMenuIconsTest(
             val currentItem = optionsMenu.getItem(i)
             if (currentItem.isVisible) {
                 Assert.assertNotNull(currentItem.icon)
-                Assert.assertTrue(currentItem.icon.isVisible)
-                Assert.assertTrue(currentItem.icon.minimumWidth >= width)
+                currentItem.icon?.let { Assert.assertTrue(it.isVisible) }
```

Comment:
Is this code change necessary?
also using !! should be only used as a last resort